to-jakarta.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+# update project major version
+mvn build-helper:parse-version versions:set -DnewVersion=\${parsedVersion.nextMajorVersion}.0.0-SNAPSHOT
+
+# move to jakarta parent
+find . -type f -name 'pom.xml' -exec sed -i '' 's/smallrye-parent/smallrye-jakarta-parent/g' {} +
+
+# move versions that support jakarta
+mvn versions:set-property -Dproperty=weld.version -DnewVersion=4.0.1.SP1
+mvn versions:set-property -Dproperty=microprofile-reactive-messaging.version -DnewVersion=3.0-RC2
+mvn versions:set-property -Dproperty=microprofile-reactive-streams.version -DnewVersion=3.0-RC1
+mvn versions:set-property -Dproperty=microprofile-config.version -DnewVersion=3.0.1
+mvn versions:set-property -Dproperty=microprofile-metrics-api.version -DnewVersion=4.0.1
+mvn versions:set-property -Dproperty=microprofile-health-api.version -DnewVersion=4.0
+mvn versions:set-property -Dproperty=smallrye-config.version -DnewVersion=3.0.0-RC1
```

Comment:
How is this updated? Does dependabot handle these?

Reply:
This will update the pom directly and keep the exact same pom structure and order. Once updated, dependabot will start looking into these versions (if they are not ignored in its config file)